docs/content/Getting-Started/Core/02-Create-a-project.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: Create a project
+permalink: /getting-started/core/create-a-project
+category: Getting Started
+subCategory: Core
+menuOrder: 4
+isDisableFeedbackBlock: true
+---
+
+## Scaffold a project
+
+To create a new project, run the following command from your terminal:
+
+```bash
+npx cubejs-cli create my-first-cube-project
```

Comment:
I'm absolutely positive we should have no "npx" in the "getting started" sequence. My suggestion is to restore the `docker-compose.yml` file from the deleted docs page and recommend getting started with Docker Compose. It's a fully transparent and extendable way. Also, it lets us, later in docs, suggest to extend that same `docker-compose.yml` file with additional services, e.g., Cube Store.